devito/passes/iet/definitions.py

```diff
@@ -198,63 +206,36 @@ def place_definitions(self, iet, **kwargs):
         iet : Callable
             The input Iteration/Expression tree.
         """
+        # Process inline definitions
         storage = Storage()
 
-        refmap = FindSymbols().visit(iet).mapper
-        placed = list(iet.parameters)
-
         for k, v in MapExprStmts().visit(iet).items():
```

Comment:
Note for reviewers: please do appreciate the massive simplification in this method, which is finally readable